### ISSUE TITLE
Failed to execute 'getWriter' on 'WritableStream' with two subsequent write()s.

### ISSUE THREAD

**Issue by leoncity** [gohai/p5.webserial#8]
I would like to transmit two reasonable small arrays (6 bytes each) over a 115200 bps connection.

If I use one `write()` call, it works. If I use two `write()` calls after eachother, this error occurs:
```
p5.webserial.js:740 Uncaught (in promise) TypeError: Failed to execute 'getWriter' on 'WritableStream': Cannot create writer when WritableStream is locked
    at p5.WebSerial.write (p5.webserial.js:740:41)
    at Grid.show (grid.js:142:21)
    at draw (sketch.js:126:8)
```
It seems that calling `write()` while the old data is still being written, fails.

My draw() is only called with frame rate of 1 Hz.
```
function test() {
  if (this.serialport === null) return;
  if (!this.serialport.opened()) return;
  this.serialport.write([0xFF, 0x00, 0xFC, 0x02, 0x05, 0x0A]);
  this.serialport.write([0xFF, 0x00, 0xFC, 0x02, 0x05, 0x0A]);
}

function draw() {
  background(220);

  // changes button label based on connection status
  if (port && !port.opened()) {
    connectBtn.html('Connect to Dongle');
  } else if (port) {
    connectBtn.html('Disconnect');
  }

  if (true) {
    test();
    return;
  }
  ```

**Comment by gohai** [comment]
Thanks for your report, @leoncity. I wrote a fix for this (https://github.com/gohai/p5.webserial/commit/bc0d5120da9bdbbc3fb6d406b718f8ba5b5966e9) - will try to test it tomorrow.

**Comment by gohai** [comment]
@leoncity Fix pushed to npm as verson 1.0.7.

**Comment by leoncity** [comment]
Thanks @gohai for creating the fix. (I hadn't had time to test the fix yet.)
